### PR TITLE
CI: GitHub Actions

### PR DESCRIPTION
**Pull request by dgasmith**
## Description
Switches CI over to GitHub Actions.

## Status
- [x] Changelog updated
- [x] Ready to go

### CODE REVIEW

**Comment by dgasmith** [comment]
This is ready for review, I don't think the checks come in until its in master. Builds here: https://github.com/dgasmith/QCElemental/commit/60ee4d5ab795e7a237b0d4d49c5d18a87f639f6b/checks?check_suite_id=336612400

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
 
 **To Reproduce**
-Code to reproduce the behavior:
-
+<!-- Code to reproduce the behavior: -->
```
Comment (loriab) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "Please include the results of `conda list` and `python -c "import qcelemental as qc; print(qc.__file__, qc.__version__)"`.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate test
+        conda list --show-channel-urls
+
+    - name: PyTest
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate test
+        pytest -v --cov=qcelemental --cov-report=xml qcelemental/tests/
```
Comment (loriab) [inline .github/workflows/CI.yml]: try adding `--color yes` for quick id of failed

Comment (dgasmith) [reply .github/workflows/CI.yml]: Color worked!

**Pull request merged**